### DISCUSSION TITLE
APFS-FUSE

### DISCUSSION THREAD

**Discussion by CodeRunner5235** [dokan-dev/dokany#1112] [Q&A]
So, I dual boot MacOS Monterey on my ryzen system and my issue stems from being unable to access the apfs drive on my windows side. Now, I know there are paid options to get this to work, but I'm cheap and those options are no fun. So, I wanted to try and get this apfs-fuse driver to work with dokany on windows. Unfortunately I dont know if this is possible, as this driver was originally written for linux. So before i waste hours of my life, I wanted to ask if it is even possible to get this to work. 
apfs-fuse:  https://github.com/sgan81/apfs-fuse

**Comment by Liryna** [comment]
Hi @CodeRuner5235 ,

There is a discussion open in the repository you linked for Windows support https://github.com/sgan81/apfs-fuse/issues/78
